packages/tasks/src/tasks/image-to-image/inference.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * Inference code generated from the JSON schema spec in ./spec
+ *
+ * Using src/scripts/inference-codegen
+ */
+
+export type ImageToImageOutput = unknown[];
+
+/**
+ * Inputs for Image To Image inference
+ */
+export interface ImageToImageInput {
+	/**
+	 * One or more images to generate images from
+	 */
+	inputs: unknown;
+	/**
+	 * Additional inference parameters
+	 */
+	parameters?: unknown;
```

Comment:
FYI there are some params only from diffusers here https://github.com/huggingface/api-inference-community/blob/main/docker_images/diffusers/app/pipelines/image_to_image.py

* num_inference_steps
* negative_prompt
* guidance_scale

Reply:
Thanks for the pointer!